### PR TITLE
Make guestbook-go example use slaves

### PR DESCRIPTION
**Pull request by invino4**
The original PHP guestbook example sets up a single redis master and a set of redis slaves that mirror the master.  The guestbook web site then uses the slaves for read operations and the master only for writes.  In the port to Go the use of the slaves was inadvertently dropped.  This change restores their use in a manner similar to the PHP version.

### CODE REVIEW

File: examples/guestbook-go/_src/main.go [hunk 1]
```diff
@@ -27,11 +27,12 @@ import (
 	"github.com/xyproto/simpleredis"
 )
 
-var pool *simpleredis.ConnectionPool
+var masterPool *simpleredis.ConnectionPool
```
Comment (roberthbailey) [inline examples/guestbook-go/_src/main.go]: This should be written as

var(
  masterPool *simpleredis.ConnectionPool
  slavePool *simpleredis.ConnectionPool
)

**Comment by roberthbailey** [comment]
Thanks for fixing this!

One minor nit but otherwise LGTM. I can rebuild the docker images for guestbook-go once this is merged.

**Comment by invino4** [comment]
done.

**Comment by roberthbailey** [comment]
Thanks Jason. Can you squash your two commits and then I'll merge.

**Comment by invino4** [comment]
done.  Thanks for the help!

**Comment by roberthbailey** [comment]
Will merge on green.

**Pull request merged**